lib/modules/search.js

```diff
@@ -248,7 +248,7 @@ function onSearchResultSelected(result) {
 export function renderSearchForm(container: HTMLElement) {
 	$(container).find('#SearchRES-input-container').replaceWith(searchFormTemplate());
 
-	$(container).on('submit', '#SearchRES-input-container', e => {
+	$(container).on('keyup submit', '#SearchRES-input-container', e => {
```

Comment:
Could you add a throttle around the event handler to avoid thrashing lower-performance computers?